src/audio/CMakeLists.txt

```diff
@@ -207,7 +207,7 @@ set(volume_sources module_adapter/module_adapter.c module_adapter/module/generic
 set(mixer_sources ${mixer_src})
 set(src_sources src/src.c src/src_generic.c)
 set(asrc_sources asrc/asrc.c asrc/asrc_farrow.c asrc/asrc_farrow_generic.c)
-set(eq-fir_sources eq_fir/eq_fir.c eq_fir/eq_fir_generic.c)
+set(eq-fir_sources module_adapter/module_adapter.c module_adapter/module/generic.c eq_fir/eq_fir.c eq_fir/eq_fir_generic.c)
```

Comment:
I think I already asked about this, why does this need module_adapter sources? Shouldn't module adapter be a library that components depending on it select in kconfig?

Reply:
@singalsu This is required by testbench, without it the testbench can't find module adapter symbol.  I don't add module adapter in src/audio/eq_fir/CMakelist. please check https://github.com/thesofproject/sof/pull/5926 （Convert the EQ IIR component to use the module interface）， it also did this.

Reply:
Thanks for explanation, it's OK then. I wonder what happens if there are in testbench pipeline multiple components with duplicated module adapter? Maybe there's no conflict since they are in their own shared libraries. But not an issue for this PR.